build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -731,21 +731,31 @@ message ExecutedActionMetadata {
 
 // An ActionResult represents the result of an
 // [Action][build.bazel.remote.execution.v2.Action] being run.
+//
+// The server MUST NOT make any inlining decisions on his own but should
```

Comment:
Can we remove this comment? It's documented on the fields below, which seems like the appropriate place for it; I'm not sure that message-level comment adds anything.

Also, the server is free to *not* inline, so this one as written seems overly strong.

Reply:
done